docs/source/use_cases/discover_google_data/1000_genomes_ld.rst

```diff
@@ -0,0 +1,25 @@
+1,000 Genomes Phase 3 Linkage Disequilibrium Data
+=================================================
+
+.. comment: begin: goto-read-the-docs
+
+.. container:: visible-only-on-github
+
+   +-----------------------------------------------------------------------------------+
+   | **The properly rendered version of this document can be found at Read The Docs.** |
+   |                                                                                   |
+   | **If you are reading this on github, you should instead click** `here`__.         |
+   +-----------------------------------------------------------------------------------+
+
+.. _RenderedVersion: http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_google_data/1000_genomes_ld.html
```

Comment:
The updates look nice, but when I click on the following link it says *"SORRY This page does  not exist yet"* - is there a certain period of time I should wait?

http://googlegenomics.readthedocs.org/en/latest/use_cases/discover_google_data/1000_genomes_ld.html

Thanks,
~p